src/bots/makerBidAskTwapCrank.ts

```diff
@@ -389,7 +432,7 @@ export class MakerBidAskTwapCrank implements Bot {
 					direction: PositionDirection.LONG,
 					slot: this.latestDlobSlot!,
 					oraclePriceData,
-					numMakers: 5,
+					numMakers: 2,
```

Comment:
@crispheaney @0xbigz any issues here for reducing numMakers 5 -> 2?

Reply:
we might be able to do 4 now w new program change?

also:
1) should probably slice after concatenatedList so that we include the correct amount
2) should make sure we dont pass in user stats